mlxtend/classifier/stacking_classification.py

```diff
@@ -112,14 +120,27 @@ def fit(self, X, y):
             if self.verbose > 1:
                 print(_name_estimators((clf,))[0][1])
 
-            clf.fit(X, y)
+            # Extract fit_params for clf
+            clf_fit_params = {}
+            for key, value in six.iteritems(fit_params):
+                if name in key and 'meta-' not in key:
+                    clf_fit_params[key.replace(name+'__', '')] = value
```

Comment:
```
for key in list(six.iterkeys(fit_params)):
     if name in key and 'meta-' not in key:
          clf_fit_params[key[len(name) + 2: ]] = fit_params.pop(key)
```
might be more efficient since the `fit_params` will be iterated again.

Reply:
I suppose `key.startswith(name)` is  `True`. Is that the case?